lms/templates/components/card/card.underscore

```diff
@@ -2,40 +2,40 @@
 <div class="wrapper-card-core">
     <div class="card-core">
         <% if (pennant) { %>
-            <small class="card-type"><%- pennant %></small>
+            <small class="card-type"><%= HtmlUtils.escape(pennant) %></small>
         <% } %>
         <h3 class="card-title"
             <% if (!_.isUndefined(srInfo)) { %>
                 aria-describedby="<%= srInfo.id %>"
             <% } %>
-            ><%- title %>
+            ><%= HtmlUtils.escape(title) %>
```

Comment:
@robrap To be clear, I wasn't proposing that we never use ``<%-`` (although I can see @cahrens' point). The challenge I was pointing out is that it isn't always clear in the template whether the parameter being passed in might have already created an HTML snippet. As I was making these changes, I was running into places where folks were putting HTML into seemingly non-HTML parameters such as title.

As you suggest, it is always safe from an XSS point of view to use ``<%-`` so you just have to be aware that the parameter must not have been escaped in JavaScript. In general, I think we were suggesting that it is better to leave the escaping until the point of usage, so in most cases the parameter will be unescaped. The use case for pre-escaping is if the JavaScript wishes to create HTML snippets, so for such parameters a template needs to use ``<%= HtmlUtils.escape(...) %>". They don't strictly have to escape again, but then your linter might object to something like ``<%= messageHtml %>``, and also the template author won't be able to guarantee that there isn't a client that forgot to escape.

This is all a rather verbose way of saying that I vote for your proposal, @robrap. I don't want to update all of our templates that are already safely using ``<%-`` to use a more verbose syntax that doesn't add any extra value.